src/librustc_codegen_llvm/attributes.rs

```diff
@@ -263,7 +263,7 @@ pub fn from_fn_attrs(cx: &CodegenCx<'ll, 'tcx>, llfn: &'ll Value, instance: ty::
     // Windows we end up still needing the `uwtable` attribute even if the `-C
     // panic=abort` flag is passed.
     //
-    // You can also find more info on why Windows is whitelisted here in:
+    // You can also find more info on why Windows is allowlisted here in:
```

Comment:
This is another case where I think we could be more precise. Can we say:

> You can also find more info on why Windows always requires uwtables here:

I don't really see this as a very clear instance of a allow/deny list or what have you.